fast/stages/2-networking-b-nva/variables.tf

```diff
@@ -111,6 +113,17 @@ variable "gcp_ranges" {
   }
 }
 
+variable "network_mode" {
+  description = "Selction of the network mode to deploy"
+  type        = string
+  default     = "simple"
+  nullable    = false
+  validation {
+    condition     = contains(["simple", "ncc_ra", "gcve"], var.network_mode)
```

Comment:
this is mixing support for a product (gcve) with a network design switch; we need to find a better option IMHO

Reply:
I agree, this is a "regional VPC" networking design, there's nothing really gcve specific.

So, `regional_vpc`?

Reply:
renamed in regional_vpc